basebuild/build.go

```diff
@@ -31,7 +31,7 @@ import (
 	_ "github.com/ottemo/commerce/app/actors/xdomain"      // XDomain support module
 
 	_ "github.com/ottemo/commerce/app/actors/payment/authorizenet" // Authorize.Net payment method
-	_ "github.com/ottemo/commerce/app/actors/payment/braintree"    // Braintree payment method
+	// _ "github.com/ottemo/commerce/app/actors/payment/braintree"    // Braintree payment method
```

Comment:
we may want to remove all payment methods except for Stripe and PayPal. I think Stripe now supports paypal within it API...